### PR TITLE
[kotlin/ko-kr] Add korean translation for kotlin

### PR DESCRIPTION
**Pull request by skyisle**
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by skyisle** [comment]
@vendethiel Please review this.

File: ko-kr/kotlin-kr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+fun useObject() {
+    ObjectExample.hello()
+    val someRef: Any = ObjectExample // 객체의 이름을 그대로 사용합니다.
+}
+
+```
+
+### 더 알아보기
+
+* [Kotlin tutorials](https://kotlinlang.org/docs/tutorials/)
```
Comment (vendethiel) [inline ko-kr/kotlin-kr.html.markdown]: please add a " (EN)" tag or use kr links

Comment (skyisle) [inline ko-kr/kotlin-kr.html.markdown]: Added

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**